src/extension.ts

```diff
@@ -169,6 +169,8 @@ function findTypeScriptLocation() {
   }
 }
 
+let languageClient: LanguageClient;
+
 export function activate(context: VSCode.ExtensionContext) {
   let serverOptions = () => runJavaServer(context);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Identifier 'serverOptions' is never reassigned; use 'const' instead of 'let'. ([typescript:S3353](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3353))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&pullRequest=16&issues=AWMHLCjyaT4gCmfv8i_2&open=AWMHLCjyaT4gCmfv8i_2)

[ISSUE_KEY]: AWMHLCjyaT4gCmfv8i_2